.travis.yml

```diff
@@ -7,8 +7,8 @@ jobs:
     script:
     - export TAG=$(if [ "x$TRAVIS_TAG" != "x" ]; then echo "$TRAVIS_TAG"; else git
       rev-parse --short HEAD; fi)
-    - docker build -t $DOCKER_REPO/base base
-    - docker build -t $DOCKER_REPO/single-user single-user
+    - travis_wait 40 docker build -t $DOCKER_REPO/base base
```

Comment:
Maybe too much: 
`The job exceeded the maximum time limit for jobs, and has been terminated.`

https://travis-ci.org/github/EGI-Foundation/egi-notebooks-images/builds/672567955#L816

Reply:
it takes forever to build these images, it's timing out constantly. I think there needs to be a better way to build them, but still cannot figure out how. 

This: https://travis-ci.org/github/jupyter/docker-stacks builds  ~8 images and just takes 30 minutes on travis 🤔

Reply:
Maybe they are paying?
Also last week/over the week end there was troubles with Travis, so for a personal project I moved to Circle CI, we may give it a try. In fact we could also try to use GitHub Actions, may even be simpler.

Reply:
I don't think travis is the main issue. 

Conda gets extremely slow for these builds. Specially when adding bioconda, which turns it unusable. Same thing when adding the onedata packages. 

I'm trying mamba https://github.com/QuantStack/mamba, which seems way faster.

Reply:
> Still in BETA! Don't use mamba to install something in valuable environments!
But if it's fine with you it's fine with me :)

Reply:
it's beta, but it seems to work and is blazingly fast. So I'm all in :)